build.sbt

```diff
@@ -317,6 +317,7 @@ val micrositeSettings = Seq(
   micrositeBaseUrl := "",
   micrositeDocumentationUrl := "/latest/api",
   micrositeHighlightTheme := "color-brewer",
+  micrositeGitterChannel := false,
```

Comment:
I have disabled this because it keeps pointing to the `scanamo/scanamo` gitter channel, which does not exist yet.